calico_kubernetes/logutils.py

```diff
@@ -3,7 +3,7 @@
 import os
 import sys
 
-LOG_DIR = '/var/log/calico/kubernetes'
+LOG_DIR = '/var/log/calico/kubernetes/exec-plugin/'
```

Comment:
Yep - if we're just using a RotatingFileHandler let's keep the same directory structure.